airflow/ui/openapi-gen/requests/types.gen.ts

```diff
@@ -688,6 +688,47 @@ export type FastAPIAppResponse = {
   [key: string]: unknown | string;
 };
 
+/**
+ * DAG Run model for the Grid UI.
+ */
+export type GridDAGRunwithTIs = {
+  run_id: string;
```

Comment:
```suggestion
  dag_run_id: string;
```

Let's try to be more consistent with the DagRunResponse

Reply:
Updated using aliases, thanks!